### ISSUE TITLE
coerce option as object doesn't work.

### ISSUE THREAD

**Issue by FGTgb** [buglabs/node-xml2json#89]
Hello,

I'm trying to specify a coercion on a specific xml attribute, but what I get is the following:

ERROR
ValidationError: child "coerce" fails because ["coerce" must be a boolean]
    at Object.exports.process (xml2json/node_modules/joi/lib/errors.js:140:17)
    at internals.Any._validateWithOptions (xml2json/node_modules/joi/lib/any.js:649:25)
    at root.validate (xml2json/node_modules/joi/lib/index.js:102:23)
    at Object.module.exports [as toJson] (xml2json/lib/xml2json.js:152:26)

The documentation reads:
coerce: Makes type coercion. i.e.: numbers and booleans present in attributes and element values are converted from string to its correspondent data types. Coerce can be optionally defined as an object with specific methods of coercion based on attribute name or tag name, with fallback to default coercion.

And test sample gives:
var result = parser.toJson(data, {reversible: true, coerce: {text:String}, object: true});

When I try this, I have the above error.
Also I've tried:
var result = parser.toJson(data, {reversible: true, coerce: {'MyAttributeName':String}, object: true});
but it doesn't work either.

Thanks for any help.

FGTGB

**Comment by c4milo** [comment]
@danielb2, @ZJONSSON any idea of what might be happening here?

**Comment by danielb2** [comment]
`ValidationError: child "coerce" fails because ["coerce" must be a boolean]`

It looks like you're trying to set `coerce` to be a string.

The default values are:

``` javascript
var options = {
    object: false,
    reversible: false,
    coerce: true,
    sanitize: true,
    trim: true,
    arrayNotation: false
};

```

**Comment by c4milo** [comment]
@danielb2 I'm asking because you were pushing for having https://github.com/buglabs/node-xml2json/pull/80 merged, which adds a new feature: being able to pass objects to the coerce property.

**Comment by danielb2** [comment]
Oh you're right. coerce needs to be able to take a boolean or an object. My bad. I'll fix. Thank you

**Comment by c4milo** [comment]
@danielb2 no problem, thanks for looking into this!

**Comment by danielb2** [comment]
PR submitted.

**Comment by c4milo** [comment]
@danielb2's fix landed in v0.8.2 in NPM. Thanks @danielb2!

**Comment by FGTgb** [comment]
Hello all,

Thanks a lot for your quick support on this.
I've just tested the new version, and it works fine now: my specific attribute is properly converted to a string.

FGTgb
